core-java-modules/core-java-date-operations-3/src/main/java/com/baeldung/unixtime/UnixTimeUtils.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.unixtime;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Calendar;
+import java.util.Date;
+
+public class UnixTimeUtils {
```

Comment:
Since this a utility class, a good practice is to hide the public constructor by adding a private constructor.